src/sgame/sg_utils.cpp

```diff
@@ -451,7 +451,7 @@ bool G_ClientIsLagging( gclient_t *client )
 {
 	if ( client )
 	{
-		if ( client->ps.ping >= 999 )
```

Comment:
How does it work with bots?

Reply:
this function is only once used.
when a client with duplicate guid connects.

there is probably a problem in a different place that lagging clients have a ping of 0 but this solves the problem